cli/tests/unit/command_test.ts

```diff
@@ -424,6 +424,26 @@ Deno.test({ permissions: { run: true } }, function commandSyncNotFound() {
   );
 });
 
+Deno.test({ permissions: { run: true } }, function cwdNotFound() {
+  assertThrows(
+    () => new Deno.Command("ls", {
+      cwd: "/sadsa"
+    }).output(),
+    Deno.errors.NotFound,
+    "Can't find cwd path `/sadsa`: No such file or directory (os error 2)"
```

Comment:
I'm not sure the message will be the same on every operating system and it might be good to test a path that would work on windows. So for example, we could use use `Deno.cwd() + "/non-existent-path"` and then just check that the error message contains "non-existent-path".

Reply:
Wouldn't it be better to do a check for only "Not a directory" or "No such file or directory" instead of the path name

Reply:
Each operating system has a slightly different message and Windows uses backslashes in paths. This is already checking for `Deno.errors.NotFound` so that's probably good enough in addition to just the filename

Reply:
Okay, I'll add the filename + this message